gp2gp-translator/src/test/java/uk/nhs/adaptors/pss/translator/mapper/medication/MedicationRequestMapperTest.java

```diff
@@ -93,9 +105,9 @@ public void When_MappingMedicationStatement_Expect_CorrectMappersToBeCalled() {
             .stream()
             .filter(resource -> ResourceType.MedicationStatement.equals(resource.getResourceType()))
             .map(MedicationStatement.class::cast)
-            .forEach(medicationStatement -> {
-                assertThat(medicationStatement.getSubject().getResource().getIdElement().getIdPart()).isEqualTo(PATIENT_ID);
-                assertThat(medicationStatement.getDateAssertedElement().getValue())
+            .forEach(it -> {
+                assertThat(it.getSubject().getResource().getIdElement().getIdPart()).isEqualTo(PATIENT_ID);
```

Comment:
In general I am more in favor of keeping everything in one place in test classes but here as the test is quite lengthy perhaps it would bring some more benefits to make the test more concise and move the last two sections into separate assertion methods like below: 
private void assertMedicationRequests(List<Resource> resources) {
        resources.stream()
            .filter(resource -> ResourceType.MedicationRequest.equals(resource.getResourceType()))
            .map(MedicationRequest.class::cast)
            .forEach(medicationRequest -> {
                assertThat(medicationRequest.getSubject().getResource().getIdElement().getIdPart()).isEqualTo(PATIENT_ID);
            });
    }